### ISSUE TITLE
A metric for star/galaxy separation

### ISSUE THREAD

**Issue by ctslater** [LSSTScienceCollaborations/ObservingStrategy#81]
Much of the Galactic structure science (with non-variable stars) will be limited in depth by our ability to separate stars from small unresolved galaxies. This is mostly dependent on seeing for morphological s/g separation and photometric precision for color-based s/g separation. We would like to be able to translate the conditions of numerous epochs into a limiting depth at which we experience e.g. 10% or 50% contamination by galaxies. 

Suggestions for how to perform this exercise are welcome. My first thought is that this will likely require modeling how we plan to perform s/g separation (psf minus aperture mag is probably not the Theoretically Optimal solution), including a realistic background population of galaxies. We will probably want individual metrics for morphology and color separation.

**Comment by ivezic** [comment]
I have image simulations (based on gaussians, but good enough for this context) that I used
to measure performance of various star-galaxy separators (including the SDSS method and
the optimal Bayesian method). I also have a decent recipe for calculating galaxy size distribution
vs. magnitude. I will cook up a metric that will give what you asked for: what fraction of galaxies
are resolved at X% confidence level as a function of magnitude SNR. This morphological result
can then be combined with color-based and other schemes. If we did this together, we'd reach 
the end result much faster!

**Comment by ctslater** [comment]
Sounds perfect, we will definitely work on this back at UW.

**Comment by dnidever** [comment]
I have some good SMASH DECam data that you could use to play around with stellar locus cuts.

**Comment by drphilmarshall** [comment]
@ctslater I think @ivezic is offering to stay up all night pair coding with you. Carpe diem! :-)

**Comment by drphilmarshall** [comment]
@ctslater @ivezic How did the star-galaxy metric writing go in the end? It seemed like this would be relevant to the WL group ( @jmeyers314 ) as well as the Milky Way satellites people ( @bethwillman ). Are there white paper sections just waiting for this metric to get finished?

**Comment by ivezic** [comment]
Yes, Colin is on top of it. We think we can have a decent metric
on timescale relevant for this white paper!

**Comment by drphilmarshall** [comment]
The primary mention of "Star / Galaxy Separation" in the current version of the white paper is in the MW Halo subsection of the Milky Way chapter's Future Work subsection, 4.4.2. @willclarkson shoudl we punt this issue to WP version 2 via the Tucson 2016 milestone, or are you hopeful of promoting the MW halo section to the results part of the chapter in the next week or two? Thanks!

**Comment by willclarkson** [comment]
@akvivas can comment more, but at this stage yes I think this can be punted to Tucson. I don't think that evaluation of the star/galaxy metric is likely by Friday.

**Comment by akvivas** [comment]
agree.  @ctslater

**Comment by ctslater** [comment]
Agreed. I have a metric that works, but it is basically unverified and needs to be tested against existing data. That process is going slower than I had hoped.

**Comment by drphilmarshall** [comment]
Cool! Arizona here we come :-) Good luck with the testing, Colin! We're rooting for you :-)
